core/scripts/cli/cli.js

```diff
@@ -27,6 +28,8 @@ async function topMenu() {
 }
 
 async function run() {
+  // TODO: Should do a check here to detect if contracts are deployed,
```

Comment:
Do you need to do this kind of check? Because the script runs in a truffle environment it will throw it's hands up if you don't have these artifacts. However I suppose if your web3 provider is pointing to a network that is not deployed this could cause an issue

Reply:
yeah right now it sort of just blows up if the contracts aren't deployed to the network. I do assume that the user has these artifacts though.

Reply:
again, super edge case, because in production the contracts would be deployed on mainnet, and presumably all test nets